LEAF_Request_Portal/js/formSearch.js

```diff
@@ -1413,53 +1443,30 @@ var LeafFormSearch = function (containerID) {
         if (gate === undefined) {
             gate = "AND";
         }
-        let widget =
-            '<tr id="' +
-            prefixID +
-            "widget_" +
-            widgetCounter +
-            '" style="border-spacing: 5px">\
-						<td id="' +
-            prefixID +
-            "widgetRemove_" +
-            widgetCounter +
-            '"><button id="widgetRemoveButton"><img src="' +
-            rootURL +
-            'dynicons/?img=list-remove.svg&w=16" style="cursor: pointer" alt="remove search term" tabindex="0"></button></td>\
-						<td style="text-align: center"><strong id="' +
-            prefixID +
-            "widgetGate_" +
-            widgetCounter +
-            '" value="' +
-            gate +
-            '">' +
-            gate +
-            '</strong></td>\
-						<td><select id="' +
-            prefixID +
-            "widgetTerm_" +
-            widgetCounter +
-            '" style="width: 150px" class="chosen" aria-label="condition">\
-            				<option value="title">Title</option>\
-            				<option value="serviceID">Service</option>\
-            				<option value="dateSubmitted">Date Submitted</option>\
-            				<option value="categoryID">Type</option>\
-            				<option value="userID">Initiator</option>\
-            				<option value="dependencyID">Requirement</option>\
-            				<option value="stepID">Current Status</option>\
-            				<option value="data">Data Field</option>\
-            				</select></td>\
-			            <td id="' +
-            prefixID +
-            "widgetCondition_" +
-            widgetCounter +
-            '"></td>\
-						<td id="' +
-            prefixID +
-            "widgetMatch_" +
-            widgetCounter +
-            '"></td>\
-					  </tr>';
+        let widget = `<tr id="${prefixID}widget_${widgetCounter}" style="border-spacing: 5px">
+                <td id="${prefixID}widgetRemove_${widgetCounter}">
+                    <button type="button" id="${prefixID}widgetRemoveButton_${widgetCounter}" aria-label="remove filter row" style="cursor: pointer">
+                        <img src="${rootURL}dynicons/?img=list-remove.svg&w=16" alt="" />
+                    </button>
+                </td>
+                <td style="text-align: center">
+                    <strong id="${prefixID}widgetGate_${widgetCounter}" value="${gate}">${gate}</strong>
+                </td>
+                <td>
+                    <select id="${prefixID}widgetTerm_${widgetCounter}" style="width: 150px" class="chosen" aria-label="condition">
+                        <option value="title">Title</option>
```

Comment:
Recommend rebasing on the latest RC (rc/2023-11-14/Sprint-65-c1). the order of this dropdown was recently changed, and as well as other default selections.